src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -24,6 +24,15 @@ storiesOf('JsonSchemaViewer', module)
       hideTopBar={boolean('hideTopBar', false)}
       shouldResolveEagerly={boolean('shouldResolveEagerly', false)}
       onGoToRef={action('onGoToRef')}
+      context={select(
```

Comment:
Hmmm, it doesn't make much sense from the general audience perspective.
I'd rather pass a custom tree instance that filters out what's needed and skip iteration of irrelevant nodes.
https://github.com/stoplightio/json-schema-viewer/blob/beta/src/components/JsonSchemaViewer.tsx#L41
We'd still need to figure out the UI part of the PR, but one could potentially use `rowRenderer` same as we do in case of masking.

Either way, we can do it later once a proper dialect support is to be implemented, so can leave it as is.

Reply:
> t doesn't make much sense from the general audience perspective.

To be more precise here, I should say that "context" feels sort of weird here.
If I were an external developer, I wouldn't be able to figure out what that prop does.
Finding a nicer name might help, as @marcelltoth noticed.
In any case, the change I proposed is out of scope for this PR, and I'm fine with the current shape of this PR (with the exception of the below if statement).